andela/wg-7-horsemen
# 152919279 Allow comparison of members.

### What does this PR do?
- Implements the feature enabling the users to compare amongst themselves.

### Description of task to be completed 
- Add a template for the user to compare himself to other users in the gym basing on the weight values entered.
- Add function to function to fetch weight values of different users and also to create graphs showing the various changes in weight with time.

### How should this be manually tested?
- On the main menu click on the item on the far right of the screen, on the drop down select comparisons. On this page you should view a graph showing your weights overtime, you can as well compare yourself with a member by selecting a name from the options beside the graph.
- A user should have weight values entered periodically to view the variations.

### What are the relevant pivotal tracker stories?
- https://www.pivotaltracker.com/story/show/152919279
